src/openrct2/sprites.h

```diff
@@ -1169,6 +1169,8 @@ enum
     SPR_G2_END = SPR_G2_EMPTY + 1,
 
     SPR_CSG_BEGIN = SPR_G2_END,
+    SPR_RIDE_DESIGN_PREVIEW_CLASSIC_WOODEN_ROLLER_COASTER_TRACK = SPR_CSG_BEGIN + 54744,
```

Comment:
We’ll need g2 sprites for those, because the preview would otherwise not be correct.

Reply:
Should I copy the CSG sprites to G2 and ignore the SCG sprites?

Reply:
Fixed it myself.